### PR TITLE
ICP-7382 Add fingerprint for newest Iris Contact sensor

### PR DESCRIPTION
**Pull request by Automated-House**
Added fingerprint for new Iris Contact sensor
Raw Description	01 0104 0402 00 09 0000 0001 0003 0020 0402 0500 0B05 FC01 FC02 02 0003 0019
Data	
application: 00
endpointId: 01
manufacturer: iMagic by GreatStar
model: 1116-S

### CODE REVIEW

**Review by tpmanley — DISMISSED** [review]
Thanks for the PR @prjct92eh2 ! I bought one of these the other day but hadn't had a chance to try it out until now. With your change the device fingerprinted fine and the sensor functions all worked

File: devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy [hunk 1]
```diff
@@ -34,6 +34,7 @@ metadata {
 		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3323-G", deviceJoinName: "Centralite Micro Door Sensor"
 		fingerprint inClusters: "0000,0001,0003,0402,0500,0020,0B05", outClusters: "0019", manufacturer: "Visonic", model: "MCT-340 E", deviceJoinName: "Visonic Door/Window Sensor"
 		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,0B05", outClusters: "0019", manufacturer: "Ecolink", model: "4655BC0-R", deviceJoinName: "Ecolink Door/Window Sensor"
+		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,0B05", outClusters: "0019", manufacturer: "iMagic by GreatStar", model: "1116-S", deviceJoinName: "Iris Contact Sensor"
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: Can you add the clusters "FC01,FC02" to the `inClusters` list and "0003" to the `outClusters` list so they include all the devices supported by the device?

Comment (Automated-House) [reply devicetypes/smartthings/smartsense-open-closed-sensor.src/smartsense-open-closed-sensor.groovy]: Done!

**Review by tpmanley — APPROVED** [review]
Thanks!

**Pull request merged**